recipes/djinni-support-lib/1.x.x/conanfile.py

```diff
@@ -79,12 +88,20 @@ def _supported_compilers(self):
         }
 
     def validate(self):
```

Comment:
As suggested in https://github.com/conan-io/conan-center-index/pull/5620/files#r638774704:
```suggestion
    def package_id(self):
        del self.options.target

    def validate(self):
```

Reply:
we have already discussed that this does not work because target can still be, used
implementing the suggested change will break the recipe, please see https://github.com/conan-io/conan-center-index/pull/5620#issuecomment-850740993 for details

Reply:
@a4z thanks for the reply. Sorry I got confused by that comment in the conversation